### PR TITLE
Add letter opener gem to make testing emails easier

### PR DESCRIPTION
**Pull request by murny**
First step of many for doing the request forms:

Brings in this gem: https://github.com/ryanb/letter_opener

Email in development environment now opens a nice new tab displaying the email that was sent out:
image

Instead of hiding the contents of the email inside the console.

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -37,9 +37,8 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
   #
-  config.action_mailer.delivery_method = :sendmail
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
```
Comment (pgwillia) [inline config/environments/development.rb]: The deliveries lines in this config probably don't make sense?
```suggestion

```

File: config/environments/development.rb [hunk 2]
```diff
@@ -37,9 +37,8 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
   #
-  config.action_mailer.delivery_method = :sendmail
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = true
```
Comment (pgwillia) [inline config/environments/development.rb]: The deliveries lines in this config probably don't make sense?
```suggestion
```

File: config/environments/development.rb (hunk 1 above)
Comment (murny) [reply config/environments/development.rb]: Should probably be set to false, think this is the default setting from a new rails project. Can make the change.

File: config/environments/development.rb (hunk 2 above)
Comment (murny) [reply config/environments/development.rb]: This is valid, we do want to send out email in development environment (it just sends out to letter opener gem)

**Pull request merged**